framework/src/util/CFRLogger.cpp

```diff
@@ -119,6 +128,20 @@ namespace cppmicroservices
             }
         }
 
+	std::shared_ptr<cppmicroservices::logservice::Logger>
+        CFRLogger::getLogger(const std::string& name) const
+	{
+	     auto currLogger = std::atomic_load(&logService);
```

Comment:
a crash will happen if `currLogger` is `nullptr`. This needs to be handled gracefully.